src/Controller/JobController.php

```diff
@@ -39,4 +39,60 @@ public function discoverJobs() {
     return $this->redirect('entity.ultimate_cron_job.collection');
   }
 
+  /**
+   * Displays a detailed cron job logs table.
+   *
+   * @param \Drupal\ultimate_cron\Entity\CronJob $ultimate_cron_job
+   *   The cron job which will be run.
+   *
+   * @return array
+   *   A render array as expected by drupal_render().
+   */
+  public function showLogs(CronJob $ultimate_cron_job) {
+
+    $rows = array();
+    $header = array(
+      $this->t('Severity'),
+      $this->t('User'),
+      $this->t('Name'),
+      $this->t('Lid'),
+      $this->t('Start Time'),
+      $this->t('End Time'),
+      $this->t('Init Message'),
+      $this->t('Message'),
+      $this->t('Duration'),
+    );
+
+    $log_entries = $ultimate_cron_job->getLogEntries();
+    //debug($log_entries->getData());
+    foreach ($log_entries as $log_entry) {
+      $row = array();
+      $row[] = array(
+        'data' => array(
+          '#type' => 'item',
+          '#title' => $log_entry->formatSeverity()[0]['#theme'],
```

Comment:
Try displaying this in the same way as CronJobListBuilder does:

        list($status, $title) = $log_entry->formatSeverity();
        $title = $log_entry->message ? $log_entry->message : $title;

      $row['status'] = [
        'data' => $status,
        'title' => strip_tags($title),
      ];
    }